azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/artifact/ArtifactHandlerImplV2.java

```diff
@@ -31,7 +37,14 @@
     private static final int MAX_RETRY_TIMES = 3;
     private static final String ALWAYS_DEPLOY_PROPERTY = "alwaysDeploy";
 
-    public static final String RENAMING_MESSAGE = "Renaming %s to %s";
+    private static final String WEB_CONFIG = "web.config";
+    private static final String RENAMING_MESSAGE = "Renaming %s to %s";
+    private static final String RENAMING_FAILED_MESSAGE = "Failed to rename artifact to %s, which is required in Java SE environment, " +
+            "please refer https://docs.microsoft.com/en-us/azure/app-service/containers/configure-language-java#set-java-runtime-options for more information.";
+    public static final String NO_EXECUTABLE_JAR = "No executable jar found in <resources>, please check the configuration";
```

Comment:
No executable jars are found in target folder according to resource filter %s, please make sure the resource filter is correct and you have built the jar.

Reply:
Shall we show `<resources>` information in console with xml format? Or just shows "No executable jar found in target folder according to resource filters specified in <resources>, please make sure the resource filter is correct and you have built the jar."

Reply:
Print user's value in xml or default value for resource filter.

Reply:
Print it with Resource.toString() method, as ArtifactHandlerImplV2 can't access the origin pom.xml , here is an example
`
No executable jar found in target folder according to resource filter 'Resource {targetPath: D:\Personal\WebApp\spring-petclinic\target\azure-webapp\spring-petclinic-webconfig-ab78e30d-b3c4-4e27-9f32-afd6476cfeaa, filtering: false, FileSet {directory: D:\Personal\WebApp\spring-petclinic/target, PatternSet [includes: {*.exec}, excludes: {}]}},Resource {targetPath: D:\Personal\WebApp\spring-petclinic\target\azure-webapp\spring-petclinic-webconfig-ab78e30d-b3c4-4e27-9f32-afd6476cfeaa, filtering: false, FileSet {directory: D:\Personal\WebApp\spring-petclinic/target, PatternSet [includes: {*.war}, excludes: {}]}}', please make sure the resource filter is correct and you have built the jar.
`